source/_integrations/cast.markdown

```diff
@@ -16,14 +16,14 @@ You can enable the Cast integration by going to the Integrations page inside the
 
 ## Home Assistant Cast
 
-Home Assistant has its own Cast application to show the Home Assistant UI on any Chromecast device.  You can use it by adding the [Cast entity row](/lovelace/entities/#cast) to your Lovelace UI, or by calling the `cast.show_lovelace_view` service. The service takes the path of a Lovelace view and an entity ID of a Cast device to show the view on. A `path` has to be defined in your Lovelace YAML for each view, as outlined in the [views documentation](/lovelace/views/#path). The following is a full configuration for a script that starts casting the `downstairs` tab of the `lovelace-cast` path (note that `entity_id` is specified under `data` and not for the service call):
+Home Assistant has its own Cast application to show the Home Assistant UI on any Chromecast device.  You can use it by adding the [Cast entity row](/lovelace/entities/#cast) to your Lovelace UI, or by calling the `cast.show_lovelace_view` service. The service takes the path of a Lovelace view and an entity ID of a Cast device to show the view on. A `path` has to be defined in your Lovelace YAML for each view, as outlined in the [views documentation](/lovelace/views/#path). The `dashboard_path` is the part of the Lovelace UI URL that follows the defined `base_url` Typically "lovelace". The following is a full configuration for a script that starts casting the `downstairs` tab of the `lovelace-cast` path (note that `entity_id` is specified under `data` and not for the service call):
 
 ```yaml
 'cast_downstairs_on_kitchen':
   alias: Show Downstairs on kitchen
   sequence:
   - data:
-      dashboard_path: lovelace-cast-
+      dashboard_path: lovelace
```

Comment:
This is now broken. The reason why it had dashes, it that is has to contain a dash.

@taugusti @fabaff This example is now broken.

Reply:
hmm - I thought the PR was to ensure that kind of things were found before merge.

@frenck : Maybe you could provide some actual info about the dashboard_path so users do not have to be guessing...?

Btw. since the last update from Google (a few days ago), the cast only works one out of a hundred times. 
And the dashes does not seem to change anything :-(

Reply:
![image](https://user-images.githubusercontent.com/195327/82208298-90b9b100-990b-11ea-94c9-8e00b3cc896e.png)

If you have an issue with one of the core integrations, please do raise an issue on GitHub. Thanks 👍

Reply:
I will create a an issue - I just thought it was resolved because it worked a few days ago.

But - does the above mean that users have to create a separate lovelace dashboard that is called "lovelace-cast-"?

The description is unfortonately quite unclear and not in the cast-help page.